frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -742,6 +841,9 @@ export function NeuronSegmentationForm() {
           </Space>
         </>
       }
+      jobSpecificInputFields={
+        <CollapsibleEvaluationSettings isActive={useEvaluation} setActive={setUseEvaluation} />
+      }
```

Comment:
I am just guessing here: The split merger evaluation requires an annotation to work, doesn't it? And more specific a skeleton annotation is required, isn't it?

If that's so please only set the `jobSpecificInputFields` to `CollapsibleEvaluationSettings` if the user currently has an opened annotation with a skeleton annotation.
You can retrieve whether there is a skeleton tracing via: 
```javascript
  const hasSkeletonAnnotation = useSelector((state: OxalisState) => state.tracing.skeleton != null);
```